### DISCUSSION TITLE
What back-end do you use?

### DISCUSSION THREAD

**Discussion by sneakylenny** [nuxt-community/apollo-module#538] [General]
Hello guys and gals! 👋 

I'm working on my new project and I'm interested to see what back-ends, data source and/or APIs you are using.

I currently have set up a [Laravel](https://laravel.com/) application making use of the [Lighhouse](https://lighthouse-php.com/) package.

🔽 I'm curious, let me know down below 🔽

**Comment by edgarsn** [comment]
Hi.

I use https://github.com/rebing/graphql-laravel

**Comment by jumptrnr** [comment]
bit late here - but strapi
